awswrangler/distributed/datasources/parquet_datasource.py

```diff
@@ -169,7 +169,7 @@ def write_block(write_path: str, block: Block[Any]) -> str:
                     **write_args,
                 )
             # This is a change from original FileBasedDatasource.do_write that does not return paths
-            return write_path
+            return f"s3://{write_path}"
```

Comment:
hmm good catch, the part I don't understand though is why the input write path to `write_block` is different between this ParquetDatasource class and the PandasTextDatasource in the first place? In both cases the `write_path` is obtained from the `block_path_provider` which should be the same in both implementations... Any idea @kukushking? It would be better to solve the root cause rather than fix with a prefix

Perhaps it's due to [this](https://github.com/aws/aws-sdk-pandas/blob/release-3.0.0/awswrangler/distributed/datasources/parquet_datasource.py#L142) missing in the PandasText implementation?